README.md

```diff
@@ -1,156 +1,103 @@
+# Sigma - Generic Signature Format for SIEM Systems
+
 [![sigma build status](https://github.com/SigmaHQ/sigma/actions/workflows/sigma-test.yml/badge.svg?branch=master)](https://github.com/SigmaHQ/sigma/actions?query=branch%3Amaster)
 
-![sigma_logo](./images/Sigma_0.3.png)
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="./images/sigma_logo_dark.png">
+  <source media="(prefers-color-scheme: light)" srcset="./images/sigma_logo_light.png">
+  <img alt="Shows an illustrated sun in light mode and a moon with stars in dark mode." src="./images/sigma_logo_dark.png">
+</picture>
 
-# Sigma
+<a href="https://opensourcesecurityindex.io/" target="_blank" rel="noopener">
+<img style="width: 282px; height: 56px" src="https://opensourcesecurityindex.io/badge.svg" alt="Open Source Security Index - Fastest Growing Open Source Security Projects" width="282" height="56" />
+</a>
 
-Generic Signature Format for SIEM Systems
+## Explore Sigma
 
-# What is Sigma
+To start exploring the Sigma ecosystem, please visit the official website [sigmahq.io](https://sigmahq.io)
```

Comment:
The information that it is a rule deposit is missing.
a "Welcome to the SigmaHQ rule repository" ?

Reply:
Added something that solve this. Take a look again.